.travis.yml

```diff
@@ -1,25 +1,33 @@
 language: php
 
-matrix:
-    include:
-        - php: hhvm
-        - php: 5.3
-        - php: 5.4
-        - php: 5.5
-        - php: 5.6
-        - php: nightly
-    allow_failures:
-        - php: hhvm
-        - php: nightly
-    fast_finish: true
-
 sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
 
-install: composer install -n
+matrix:
+  include:
+    - php: 5.3
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 5.6
+      env: deps=low
+    - php: 5.6
+      env: deps=high
+    - php: hhvm
+    - php: nightly
+  allow_failures:
+    - php: hhvm
+    - php: nightly
+  fast_finish: true
+
+install:
+  - composer self-update -q
+  - if [ -z "$deps" ]; then composer install -n; fi;
+  - if [ "$deps" = "low" ]; then composer update --prefer-lowest --prefer-stable --prefer-source -n; fi;
+  - if [ "$deps" = "high" ]; then composer update --prefer-source -n; fi;
```

Comment:
this can be simplified:

```yaml
matrix:
  include:
    - php: 5.6
      env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable'

install:
  - composer update $COMPOSER_FLAGS
```